money_honey_cr.py

```diff
@@ -0,0 +1,147 @@
+            if arg in ("-h", "--help"):
+                self.usage()
+                sys.exit()
+            elif arg == "--amount":
+                try:
+                    amount = Decimal(value)
+                except InvalidOperation:
+                    print("Couldn't parse amount. Make sure you are using valid number.")
+                    sys.exit(1)
+            elif arg == "--input_currency":
+                input_currency = self.check_currency(currency_changer, arg, value)
+            elif arg == "--output_currency":
+                output_currency = self.check_currency(currency_changer, arg, value)
+            elif arg == "--no_json":
+                no_json = True
+
+        if amount is None or input_currency is None:
+            print("You need to specify required parameters.\n")
+            self.usage()
+            sys.exit(2)
+        else:
+            output_string_start = "\n" + str(amount) + " " + input_currency + " is"
+            currency_changer.set_exchange_params(amount, input_currency, output_currency)
+            converted_amounts = currency_changer.get_changed_values()
+
+            if no_json:
+                if len(converted_amounts) == 1:
+                    print(output_string_start + " " + self.output_string_end(format(self.get_first_value(
+                        converted_amounts.values()), '.4f'), self.get_first_value(converted_amounts.keys())) + ".")
+                else:
+                    print(output_string_start)
+                    for currency, currency_amount in converted_amounts.items():
+                        print("\t" + self.output_string_end(format(currency_amount, '>20.4f'), currency))
+            else:
+                output_json = {}
+
+                converted_currencies = {}
+                for currency, currency_amount in converted_amounts.items():
+                    converted_currencies[currency] = format(currency_amount, '.4f')
+
+                output_json['input'] = {'amount': format(amount, '.4f'), 'currency': input_currency}
+                output_json['output'] = converted_currencies
+                print("\n" + json.dumps(output_json, indent=4))
+
+    @staticmethod
+    def get_first_value(my_list):
+        return next(iter(my_list))
+
+    @staticmethod
+    def output_string_end(converted_amount, output_currency):
+        return str(converted_amount) + " " + output_currency
+
+    def check_currency(self, currency_changer, name, currency, symbol_currencies=None):
+        """
+        Checks if currency is valid and supported.
+        :param currency_changer: Instance of CurrenciesChanger.
+        :param name: Name of the input.
+        :param currency: Currency symbol or letter code.
+        :param symbol_currencies: Array of currency codes for specific symbol.
+        :return: Check function convert_symbol for returns.
+        """
+        if symbol_currencies is not None and currency.upper() in symbol_currencies:
```

Comment:
Nice I didn't know that Python supports checking it in that way. Changed in [#6](https://github.com/ZlatyChlapec/MoneyHoney/commit/f69f0f6253e8de5851e05a24c577f10b1f4c90ae).